src/standalone/Vegetation/soil_drivers.jl

```diff
@@ -35,9 +37,10 @@ specify the soil parameters by keyword arguments.
 function PrescribedSoil(;
     root_depths::Vector{FT} = FT.(-Array(10:-1:1.0) ./ 10.0 * 2.0 .+ 0.2 / 2.0),
     ψ_soil::Function = t -> eltype(t)(FT(0.0)),
-    soil_α::FT = FT(0.2),
+    soil_α_PAR::FT = FT(0.2),
+    soil_α_NIR::FT = FT(0.2),
```

Comment:
Soil alpha NIR is usually higher than soil alpha PAR. This should be around 0.4.